manifests/integreatly-rhsso/7.6.11-1/rhsso-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,400 @@
+        - apiGroups:
+          - batch
+          resources:
+          - cronjobs
+          - jobs
+          verbs:
+          - list
+          - get
+          - create
+          - update
+          - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes/custom-host
+          verbs:
+          - create
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - list
+          - get
+          - create
+          - update
+          - watch
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          verbs:
+          - list
+          - get
+          - create
+          - update
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          - prometheusrules
+          - podmonitors
+          verbs:
+          - list
+          - get
+          - create
+          - update
+          - watch
+        - apiGroups:
+          - integreatly.org
+          resources:
+          - grafanadashboards
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resourceNames:
+          - rhsso-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - policy
+          resources:
+          - poddisruptionbudgets
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - watch
+        - apiGroups:
+          - keycloak.org
+          resources:
+          - keycloaks
+          - keycloaks/status
+          - keycloaks/finalizers
+          - keycloakrealms
+          - keycloakrealms/status
+          - keycloakrealms/finalizers
+          - keycloakclients
+          - keycloakclients/status
+          - keycloakclients/finalizers
+          - keycloakbackups
+          - keycloakbackups/status
+          - keycloakbackups/finalizers
+          - keycloakusers
+          - keycloakusers/status
+          - keycloakusers/finalizers
+          verbs:
+          - get
+          - list
+          - update
+          - watch
+        serviceAccountName: rhsso-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - Keycloak
+  - Identity
+  - Access
+  links:
+  - name: Documentation
+    url: https://access.redhat.com/products/red-hat-single-sign-on
+  - name: Keycloak
+    url: https://www.keycloak.org/
+  maintainers:
+  - email: customerservice@redhat.com
+    name: Red Hat Customer Service
+  maturity: stable
+  provider:
+    name: Red Hat
+  version: 7.6.11-opr-001
```

Comment:
more curious than anything why these versions have the `-opr-001` and `-opr-002` at the end ?

Reply:
other that that the e2e pass and can verify as soon as I have a cluster.

Reply:
nevermind the question looking at the older bundles they are all the same .